### DISCUSSION TITLE
[本地部署] 本地部署如何配置clash

### DISCUSSION THREAD

**Discussion by MossLark** [GaiZhenbiao/ChuanhuChatGPT#328] [Q&A]
clash怎么配置这个文件，在哪里
![image](https://user-images.githubusercontent.com/45580569/227122712-0c233969-9114-4ba7-afcf-18aec45cf80a.png)

**Comment by 0xkl** [comment]
同求

**Comment by woshiliduo1** [comment]
同求

**Comment by shikamaru0922** [comment]
求

**Comment by SongAoFei** [comment]
加一

**Comment by Keldos-Li** [comment]
其实只需要确保两点：

1. openai.com流量务必走代理。
2. 本地localhost或127.0.0.1等不走代理（因此，你不能设置clash为系统代理）。

我们提供的配置文件仅仅是一个示例示例示例示例示例！而且你们也将需要靠自己找到一些代理配置或机场以实现“openai.com流量走代理”。

可以参考：

![](https://user-images.githubusercontent.com/28528386/224529317-f49265d7-fb5f-4dd5-b462-930aaa0c682d.png)
*图片来源：[ChatPaper](https://github.com/kaixindelele/ChatPaper)*

**Reply by 0xkl** [reply]
如果我有机场的配置文件，但是里面没有相应配置"openai.coim"走代理，我如何在原有的配置文件里面添加让"openai.com"走代理呢。我在这边要谢谢你的回复。

**Reply by GaiZhenbiao** [reply]
可以使用Clash的Mixin功能

**Reply by SongAoFei** [reply]
没错，没错，就是要这样设置，我搞了一下午/(ㄒoㄒ)/~~

**Reply by handsomeallan** [reply]
我在win7一打开tun模式就断网，用不了，win10的那个系统可以

**Reply by handsomeallan** [reply]
> 可以使用Clash的Mixin功能

Mixin这个功能怎么用，在哪里有说明吗

**Comment by 0xkl** [comment]
如果我有机场的配置文件，但是里面没有相应配置"openai.coim"走代理，我如何在原有的配置文件里面添加让"openai.com"走代理呢。我在这边要谢谢你的回复。

**Reply by SongAoFei** [reply]
我是直接右键添加就好了，就是直接在机场配置文件上右键，然后edit
我用的是Clash
然后我发现系统代理和TUN是冲突的，只开TUN就好了

**Reply by 0xkl** [reply]
怎么做呢= =

**Reply by SongAoFei** [reply]
你直接跟之前一样正常使用吧，就把系统代理关了，只开TUN，记得Service Mode安装一下
然后看看能不能运行

**Reply by 0xkl** [reply]
我试试~

**Comment by liuyuanjun520** [comment]
> 如果我有机场的配置文件，但是里面没有相应配置"openai.coim"走代理，我如何在原有的配置文件里面添加让"openai.com"走代理呢。我在这边要谢谢你的回复。

clash的的节点右键有个rules选项，在规则中增加openai.com走代理规则

**Comment by Bigfengshan** [comment]
mac 版本的clash x 没有找到TUN，有人试过吗

**Reply by GaiZhenbiao** [reply]
应该都是有的🤔️不过我在Mac上用的是Surge

**Comment by Qwerty4401** [comment]
路由器梅林固件的Merlin Clash不支持TUN怎么搞，已设置openai.com和api.openai.com走代理的规则

**Comment by youtehub** [comment]
直接部署在Linux上呗，  [**ShellClash**](https://github.com/juewuy/ShellClash)  这上面可以直接走全代理，还不干扰本地VPN的使用，window clash 的 Tun 模式 不太好用，可能会导致window系统出现一些问题，虚拟机也可以走 **桥接模式** 直接绑定路由到外网
